src/lightning/app/utilities/cloud.py

```diff
@@ -22,10 +23,11 @@
 from lightning.app.utilities.network import LightningClient
 
 
-def _get_project(
-    client: LightningClient, project_id: str = LIGHTNING_CLOUD_PROJECT_ID, verbose: bool = True
-) -> V1Membership:
+def _get_project(client: LightningClient, project_id: Optional[str] = None, verbose: bool = True) -> V1Membership:
     """Get a project membership for the user from the backend."""
+    if project_id is None:
+        project_id = LIGHTNING_CLOUD_PROJECT_ID
```

Comment:
I'm wondering what the reason is. Is LIGHTNING_CLOUD_PROJECT_ID not a constant, being modified somewhere?